pkg/virt-launcher/virtwrap/stats/types.go

```diff
@@ -107,6 +107,8 @@ type DomainStatsNet struct {
 type DomainStatsBlock struct {
 	NameSet         bool
 	Name            string
+	AliasSet        bool
```

Comment:
Wouldn't  it be good enough to simply check if `Name` or `Alias` is empty? It hink you can get rid of both booleans, having to keep less in sync.

Reply:
+1 It seems it is used only to set the labels